contracts/TalentLayerService.sol

```diff
@@ -253,8 +255,10 @@ contract TalentLayerService is Initializable, ERC2771RecipientUpgradeable, UUPSU
         address _rateToken,
         uint256 _rateAmount,
         uint16 _platformId,
-        string calldata _dataUri
+        string calldata _dataUri,
+        uint256 _proposalTimeout
     ) public payable onlyOwnerOrDelegate(_profileId) {
+        _proposalTimeout = _proposalTimeout == 0 ? 15 days : _proposalTimeout;
```

Comment:
- the timeout should be the timestamp when the proposal will be expired (not a duration)
- we don't need any require or default value